tests/_support/AcceptanceTester.php

```diff
@@ -215,7 +215,7 @@ public function amOnOrdersPage() {
 	 */
 	public function amOnOrderPage( $order_id ) {
 
-		if ( wc_get_container()->get( CustomOrdersTableController::class )->custom_orders_table_usage_is_enabled() ) {
+		if ( class_exists( OrderUtil::class ) && OrderUtil::custom_orders_table_usage_is_enabled() ) {
```

Comment:
Similar to the above, we I think it would be acceptable to make use of the `PageController::get_edit_url()` method in this case.